README.md

```diff
@@ -65,6 +64,13 @@ For more advanced features, try the following command for help message
 onnxsim -h
 ```
 
+Or run in docker
+
+```
+docker build -t onnx-simp .
+docker run --rm -it --entrypoint pytest onnx-simp
```

Comment:
what is the use case of entry point "pytest"?

Reply:
> 

Just make sure it would work properly. There are 3 failures so far (expected, as described in comments).